src/frontend/view/view-search-panel.ts

```diff
@@ -547,7 +544,7 @@ const viewKeywordsInput = () => `
   <div class="govuk-body">
```

Comment:
```suggestion
  <div class="govuk-form-group">
```

Reply:
I can't seem to add the suggestion for the one above. Can you change    `<div class="govuk-body">`
to   `<div class="govuk-form-group">` it's the div wrapped around the label for "Search for pages that link to".